docs/en/setup/advanced/AsyncEnhancement.md

```diff
@@ -0,0 +1,70 @@
+
+- **gRPC**: We use the [`grpc.aio`](https://grpc.github.io/grpc/python/grpc_asyncio.html) module to replace the `grpc` module. Since the `grpc.aio` module is also officially supported and included in the `grpc` package, we can use it directly without any additional installation.
+
+- **HTTP**: We use the [`aiohttp`](https://github.com/aio-libs/aiohttp) module to replace the `requests` module.
+
+- **Kafka**: We use the [`aiokafka`](https://github.com/aio-libs/aiokafka) module to replace the `kafka-python` module.
+
+## Proformance improvement
+
+We using [wrk](https://github.com/wg/wrk) to pressure testing the network throughput of the Python agents in a [FastAPI](https://github.com/tiangolo/fastapi) application.
+
+- gRPC
+
+The performance has been improved by about **32.8%**
+
+|      gRPC       |   QPS   |   TPS    | Avg Latency |
+| :-------------: | :-----: | :------: | :---------: |
+| sync (original) | 899.26  | 146.66KB |  545.97ms   |
+|   async (new)   | 1194.55 | 194.81KB |  410.97ms   |
+
+- HTTP
+
+The performance has been improved by about **9.8%**
+
+|      HTTP       |  QPS   |   TPS   | Avg Latency |
+| :-------------: | :----: | :-----: | :---------: |
+| sync (original) | 530.95 | 86.59KB |    1.53s    |
+|   async (new)   | 583.37 | 95.14KB |    1.44s    |
+
+- Kafka
+
+The performance has been improved by about **89.6%**
+
+|      Kafka      |  QPS   |   TPS    | Avg Latency |
+| :-------------: | :----: | :------: | :---------: |
+| sync (original) | 345.89 | 56.41KB  |    1.09s    |
+|   async (new)   | 655.67 | 106.93KB |    1.24s    |
+
+> In fact, only the performance improvement of gRPC is of more reference value. Because the other two protocols use third-party libraries with completely different implementations, the performance improvement depends to a certain extent on the performance of these third-party libraries.
+
+More details see this [PR](https://github.com/apache/skywalking-python/pull/316) .
+
+## Potential problems
+
+We have shown that the asynchronous enhancement function improves the transmission efficiency of metrics, traces and logs. But **it improves the proformance of profile data very little, and even causes performance degradation**.
+
+This is mainly because a large part of the data in the `profile` part comes from the monitoring and measurement of Python threads, which is exactly what we need to avoid in asynchronous enhancement. Since operations on threads cannot be bypassed, we may need additional overhead to support cross-thread coroutine communication, which may lead to performance degradation instead of increase.
+
+Asynchronous enhancements involved many code changes and introduced some new dependencies. Since this feature is relatively new, it may cause some unexpected errors and problems. **If you encounter them, please feel free to contact us or submit issues and PRs**!
```

Comment:
```suggestion
Asynchronous enhancements involve many code changes and introduced some new dependencies. Since this feature is relatively new, it may cause some unexpected errors and problems. **If you encounter them, please feel free to contact us or submit issues and PRs**!
```